src/components/global/FormFieldTextRequired.vue

```diff
@@ -0,0 +1,114 @@
+<script lang="ts">
+/**
+ * FormFieldTextRequired Component
+ *
+ * The `FormFieldTextRequired` displays text required input.
+ *
+ * @description * Use this component to render text required input in forms.
+ *
+ * Note: This component is commonly used in `FormRegisterCoordinator`, `FormRegisterChallenge`.
+ *
+ * @props
+ * - `value` (string, required): The object representing user input.
+ *   It should be of type `string`.
+ *
+ * @events
+ * - `update:modelValue`: Emitted as a part of v-model structure.
+ *
+ * @example
+ * <form-field-text-required v-model="value" />
+ *
+ * @see [Figma Design](https://www.figma.com/file/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?type=design&node-id=4858%3A103756&mode=dev)
+ */
+
+// libraries
+import { defineComponent, computed } from 'vue';
+
+// composables
+import { useValidation } from 'src/composables/useValidation';
+
+export default defineComponent({
+  name: 'FormFieldTextRequired',
+  props: {
+    modelValue: {
+      type: String,
+      required: true,
+    },
+    bgColor: {
+      type: String as () => 'white' | 'transparent',
+      default: 'transparent',
+    },
+    name: {
+      type: String,
+      required: true,
+    },
+    label: {
+      type: String,
+      required: true,
+    },
+    labelShort: {
+      type: String,
+    },
+    autocomplete: {
+      type: String,
+    },
+    testing: {
```

Comment:
`testing` prop is declared, but unused inside component body.

Reply:
You are right. I have used it previously to disable `lazy-rules` for easier testing. However, the recent update to Quasar requires that we call .blur() in tests after entering value into the input, so this is no longer needed.
Removed.